# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/D3D11ShareHandleImage.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/layers/D3D11ShareHandleImage.cpp
--- a/gfx/layers/D3D11ShareHandleImage.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/layers/D3D11ShareHandleImage.cpp	Wed May 22 22:27:37 2019 +0000
@@ -10,8 +10,8 @@
 #include "WMF.h"
 #include "d3d11.h"
 #include "gfxImageSurface.h"
-#include "gfxPrefs.h"
 #include "gfxWindowsPlatform.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/gfx/DeviceManagerDx.h"
 #include "mozilla/layers/CompositableClient.h"
 #include "mozilla/layers/CompositableForwarder.h"
@@ -256,7 +256,7 @@
   mImageDevice = device;
 
   TextureAllocationFlags allocFlags = TextureAllocationFlags::ALLOC_DEFAULT;
-  if (gfxPrefs::PDMWMFUseSyncTexture() ||
+  if (StaticPrefs::PDMWMFUseSyncTexture() ||
       mDevice == DeviceManagerDx::Get()->GetCompositorDevice()) {
     // If our device is the compositor device, we don't need any synchronization
     // in practice.